web/html/src/manager/minion/proxy/proxy-config-messages.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+
+import { Messages } from "components/messages/messages";
+
+type SuccessType = boolean | undefined;
+
+export const ContainerConfigMessages = (success: SuccessType, messagesIn: React.ReactNode[], loading: boolean) => {
+  if (success) {
```

Comment:
This structure works, but feels a bit clunky with multiple branches all wrapped in a `Messages`.

Since `Messages` also returns null if there are no items, something along these lines would also work:

```tsx
let items: MessageType[] = [];
if (success) {
  items = [{
    severity: "success",
    text: <p>{t("Proxy configuration successfully applied.")}</p>,
  }];
} else if ... { }

return <Messages items={items} />;
```

What do you think?